maven-java/akkaserverless-maven-archetype-value-entity/src/main/resources/archetype-resources/pom.xml

```diff
@@ -206,7 +206,7 @@
       <plugin>
         <groupId>com.akkaserverless</groupId>
         <artifactId>akkaserverless-maven-plugin</artifactId>
-        <version>@project.version@</version>
+        <version>${D}{akkaserverless-sdk.version}</version>
```

Comment:
`${D}`?

Reply:
This is to avoid premature substitution. It's used everywhere in this file whenever we want the archetype to have a ${var}. 

However, it may not be needed. I can check it later, but don't want to change more than necessary here.

Reply:
For instance, like here:

https://github.com/lightbend/akkaserverless-java-sdk/blob/main/maven-java/akkaserverless-maven-archetype-value-entity/src/main/resources/archetype-resources/pom.xml#L11-L33

Reply:
Trixy!